packages/next-polyfill-module/src/index.js

```diff
@@ -32,7 +32,8 @@ if (!('trimEnd' in String.prototype)) {
 if (!('description' in Symbol.prototype)) {
   Object.defineProperty(Symbol.prototype, 'description', {
     get: function get() {
```

Comment:
Could you set `configrable: true` just like the native implementation and core-js polyfil? Then we'll be able to add another pollyfill for Egde 44.

```suggestion
    configurable: true,
    get: function get() {
```

Reply:
I ran to this issue so I open https://github.com/vercel/next.js/pull/18575 to add the `configurable` key.